src/pages/api/graphql.ts

```diff
@@ -72,6 +77,18 @@ export const queryResolvers: QueryResolvers<AppContext> = {
 
     return data.project.drop as Drop;
   },
+  async dropPurchases(_a, _b, { dataSources: { holaplex } }) {
```

Comment:
```suggestion
  async collectible(_a, _b, { dataSources: { holaplex } }) {
```

Return the whole drop minus the purchases then add a field resolver on the collectible object that has a field of mintHistory. Make this return the dropPurchases. Don't include the price. I'm going to replace drop.purchases with collection.mintHistory very soon. This will translate better.

Reply:
Also the collections query should be a field on the user called collectibles. Again make a resolver so its only fetched when requested.

https://github.com/holaplex/hub-starter/pull/24/files#diff-d449237cb065e6b3dca3d0519882357d394b62c4048670dd27b9839705eacf6dR105